Azael/index.js

```diff
@@ -13,12 +13,37 @@ let age = 25;
 ```
```

Comment:
It seems you forgot to answer the first exercise, there is just readme comments here!